code/src/WijmoProvider/Features/CellStyle.ts

```diff
@@ -0,0 +1,78 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace WijmoProvider.Feature {
+    export class CellStyle
+        implements
+            OSFramework.Interface.IBuilder,
+            OSFramework.Feature.ICellStyle {
+        private _grid: WijmoProvider.Grid.IGridWijmo;
+        private readonly _internalLabel = '__cellStyle';
+        private _metadata: OSFramework.Interface.IRowMetadata;
+
+        constructor(grid: WijmoProvider.Grid.IGridWijmo) {
+            this._grid = grid;
+            this._metadata = this._grid.rowMetadata;
+        }
+
+        private _formatItems(
+            grid: wijmo.grid.FlexGrid,
+            e: wijmo.grid.FormatItemEventArgs
+        ) {
+            if (
+                e.panel.cellType === wijmo.grid.CellType.Cell &&
+                this.getMetadata(e.row).cssClass.size > 0 &&
+                this.getMetadata(e.row).cssClass.get(
+                    this._grid.provider.getColumn(e.col).binding
```

Comment:
Some small optimization, as putting this in a local const would help to make the code more readable,

Reply:
done